SETUP.md

```diff
@@ -0,0 +1,171 @@
+# Setup
+
+This document provides steps that needs to be taken in order to successfully setup the infrastructure using Docker and Docker Compose.
+The infrastructure can be setup with or without Keycloak service (which provides SSO). In that matter there are two separate section 
+which describes infrastructure setup weather the Keycloak service is used.
+
+**Note**: For quick setup it is recommended to setup the infrastructure without
```

Comment:
I'll verify if these instructions are sound by going over this procedure and following it to the letter. I'll try to get around to this at the start of next week, but it might slip to the end of august because of my vacation.